pkg/feature/manifest.go

```diff
@@ -24,23 +26,34 @@ type manifest struct {
 	template,
 	patch,
 	processed bool
+	processedContent string
 }
 
-func loadManifestsFrom(path string) ([]manifest, error) {
+func loadManifestsFrom(embeddedFS embed.FS, rootPath string) ([]manifest, error) {
```

Comment:
I think we could align it with walk `fs.WalkDir` expects. We do not need anything specific from `embeddedFS` to make this whole thing work, and this is a limitation we don't need.

```suggestion
func loadManifestsFrom(fsys fs.FS, rootPath string) ([]manifest, error) {
```

note: I can't apply suggestion on the whole `func` so you will need to make changes in the editor, not here.